dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java

```diff
@@ -117,4 +119,9 @@ public void executeAction(final WorkflowProcessor processor,
             throw new WorkflowActionFailureException(e.getMessage(), e);
         }
     }
+
+    @Override
+    public boolean stopProcessing() {
```

Comment:
doc?

Reply:
I am overriding a super class method, it is doc there